poc_mobile_verifier/migrations/4_speedtests.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TYPE speedtest AS (
+       timestamp timestamp,
+       upload_speed bigint,
+       download_speed bigint,
+       latency integer
+);
+
+CREATE TABLE speedtests (
+       id test primary key not null,
```

Comment:
what is `test` here?  Did you mean `text` ?

Reply:
Yup